cmd/ci-operator/main.go

```diff
@@ -1419,3 +1437,16 @@ func getSSHSecretFromPath(sshKeyPath string) (*coreapi.Secret, error) {
 
 	return secret, nil
 }
+
+func getPullSecretFromFile(filename string) (*coreapi.Secret, error) {
+	secret := &coreapi.Secret{Data: make(map[string][]byte)}
+	secret.Type = coreapi.SecretTypeDockerConfigJson
```

Comment:
nit: this line and the following could be part of the literal initialization

Reply:
Done!